tasks/django.yml

```diff
@@ -156,3 +156,12 @@
     or "changed" in load_initial_data_result.stdout
   args:
     chdir: "{{ openwisp2_path }}"
+
```

Comment:
@Purhan you should be able to change permissions in the following code. Hence, removing the need for adding a dedicated task for it. 

https://github.com/openwisp/ansible-openwisp2/blob/766a49bdcd06775a9d333dbfea8d3019d58076d1/tasks/django.yml#L3-L8

I believe the idempotence test is also failing due to this. While running above task during idempotence test, molecule observes a change and hence the test suite fails.

Reply:
Thanks! I will quickly implement these changes. One question: the issue doesn't mention anything about write permissions, should it be granted to user?

Reply:
I have done the changes I needed locally and polished them. Will send them asap. I suggest to pick another issue, thanks!